packages/account-v2/src/containers/POAForm/DocumentSubmission/DocumentSubmission.tsx

```diff
@@ -16,33 +15,33 @@ const listItems = [
 const documentValidation = Yup.mixed().required('Please upload a file');
 
 const DocumentSubmission: React.FC = () => {
-    const { isMobile } = useBreakpoint();
+    const { isMobile } = useDevice();
 
     return (
-        <div className='flex flex-col items-start w-full gap-1200 sm:w-auto'>
-            <div className='flex h-1200 gap-400 self-stretch sm:self-auto justify-center items-center sm:gap-[11px]'>
+        <div className='flex flex-col items-start w-full gap-24 sm:w-auto'>
```

Comment:
```suggestion
        <div className='flex flex-col items-start w-full gap-24 w-auto'>
```

if only required on `sm`, please ignore this comment

Reply:
We will change the whole structure for `sm:` and `md:` later.